tests/test_util.py

```diff
@@ -201,6 +198,24 @@ def test_duplicate_expression_alias(self):
         ]
         assert exprs == ['(topK(3)(logger) AS dupe_alias)', 'dupe_alias']
 
+    def test_nested_aggregate_legacy_format(self):
+        def assert_aggregation(body, expected, index=0):
+            col, alias, agg = '', body[index][2], body[index][0]
+            body = {'aggregations': body}
+            assert column_expr(col, body, alias, agg) == expected
+
+        last_seen = [['multiply(toUInt64(max(timestamp)), 1000)', '', 'last_seen']]
+        assert_aggregation(last_seen, '(multiply(toUInt64(max(timestamp)), 1000) AS last_seen)')
+
+        first_seen = [['multiply(toUInt64(min(timestamp)), 1000)', '', 'first_seen']]
+        assert_aggregation(first_seen, '(multiply(toUInt64(min(timestamp)), 1000) AS first_seen)')
+        priority = [
+            ['count()', '', 'times_seen'],
+            last_seen[0],
+            ['toUInt64(plus(multiply(log(times_seen), 600), last_seen))', '', 'priority']
```

Comment:
Do we really need these multiple aggregations (and the `index` param to `assert_aggregation`)? Given that the test code is both creating the list, and picking one item from it to pass to `column_expr`, it's not really testing anything about the snuba code's ability to handle a list of aggregations.

Reply:
@alex-hofsteede let me know what you think of this change.

Reply:
Hmm, so does this test fail if we don't include the first 2 aggregations? I see now that the last aggregation references the aliases of the first two, but I don't remember if we would actually throw an error in the case where we were trying to use an alias that we hadn't already defined before?

Reply:
No it doesn't I can leave them out.